src/main/java/org/sonarqube/gradle/SonarPropertyComputer.java

```diff
@@ -248,8 +250,9 @@ private static void extractTestProperties(Project project, Map<String, Object> p
   private static void configureJaCoCoCoverageReport(final Test testTask, final boolean addForGroovy, Project project, final Map<String, Object> properties) {
     project.getTasks().withType(JacocoReport.class, jacocoReportTask -> {
       SingleFileReport xmlReport = jacocoReportTask.getReports().getXml();
-      if (xmlReport.isEnabled() && xmlReport.getDestination().exists()) {
-        appendProp(properties, "sonar.coverage.jacoco.xmlReportPaths", xmlReport.getDestination());
+      File reportDestination = getDestination(xmlReport);
+      if (isReportEnabled(xmlReport) && reportDestination.exists()) {
```

Comment:
check for null here too

Reply:
Already done in last commit, or am I missing something?